tekton/README.md

```diff
@@ -0,0 +1,44 @@
+# Release Script
+
+The Release script does the following tasks:
+- fetch the latest main branch
+- create and push a git tag by version provided from the user
+- create supporting configuration for hub on the cluster
+- create secret and roles for the Tekton Pipelines
+- install the pipeline which will deploy the hub on the clusters
+
+## Prerequisites
+
+- Kubernetes or OpenShift cluster with Tekton Pipelines installed
+- kubectl CLI
+- git CLI
+
+### User Prerequisites
+
+- Access to push images to `quay.io/tekton-hub`
+- Access to push git tag to `tektoncd/hub`
+
+## Before Running the Script
+
+- Check the last hub release version [here](https://github.com/tektoncd/hub/releases) and enter the next in the script to
+- If you are deploying hub on a clean cluster
+    - You will need GitHub OAuth. You can create one using the steps given [here](https://docs.github.com/en/developers/apps/creating-an-oauth-app). Use any url for creating OAuth, later you can change it once hub is deployed.
+
+
+## Running the Script
+
+This will deploy the hub in `tekton-hub` namespace and install the Tekton pipelines for it in the `tekton-hub-ci` namespace.
```

Comment:
nit: (feel free to update only if there are other changes)
```suggestion
This will deploy the hub in `tekton-hub` namespace and run the Tekton pipelines for it in the `tekton-hub-ci` namespace.
```